lib/tasks/evm_import_export_alerts.rake

```diff
@@ -0,0 +1,114 @@
+# Rake script to export Alerts and AlertSets
+
+require_relative 'evm_import_export_illegal_chars'
+require_relative 'evm_import_export_options'
+
+class EvmImportExportAlerts
```

Comment:
These classes should probably live in [lib/task_helpers](https://github.com/ManageIQ/manageiq/tree/02c1f12c0f13efeb9a3faaa8c5031a3fd5a20410/lib/task_helpers).